src/Workspaces/Core/Portable/CodeRefactorings/FixAllOccurences/FixAllContext.cs

```diff
@@ -50,7 +51,7 @@ internal sealed class FixAllContext : IFixAllContext
         /// </summary>
         public CancellationToken CancellationToken { get; }
 
-        internal IProgressTracker ProgressTracker { get; }
+        public IProgress<CodeAnalysisProgress> Progress { get; }
```

Comment:
> public IProgress<CodeAnalysisProgress> Progress { get; } [](http://example.com/codeflow?start=7&length=57)

DocComment since public now?

Reply:
Working on this.  Makign internal for now.  Intend to remove entirely.  Will involve a follow up PR.  will let you know when ready.